test/libsolidity/semanticTests/builtins/smoke.sol

```diff
@@ -0,0 +1,14 @@
+contract ClientReceipt {
```

Comment:
I think this smoke test file should not be in `semanticTests` as it is confused. All the other tests for the testparser are in test/libsolidity/util/TestFunctionCallTests.cpp.

Reply:
Theoretically we could just remove the smoke test (soon). We will have real built-ins (e.g. balance) and real test hooks (e.g. gas expectation & isoltest events) soon. The thing is that these smoke tests are not just there to test the testparser.